FeaturedProducts.md

```diff
@@ -1,11 +1,27 @@
 # Featured Products
 
-* **Regular Coffee**
- * Just your regular cup of coffee
- * _$1.99_
-* **Decaffeinatee Coffee**
- * Coffee, but with out the jitters
- * _$1.99_
-* **Espresso**
- * A fantastic Espresso for an quick "pick me up"
- * _$2.99_
\ No newline at end of file
+### Regular Coffee
+  * Just your regular cup of coffee
+  * **Pricing**
+    * _$1.99_ - Standard Price
+    * _$1.79_ - with Loyalty Discount
+    * _$1.79_ - when paying with CoffeHut Wallet
+    * _$1.49_ - Loyalty Discount & CoffeHut Wallet 
```

Comment:
```suggestion
    * _$1.79_ - with Loyalty Discount (10% Off!)
    * _$1.79_ - when paying with CoffeHut Wallet (10% Off!)
    * _$1.49_ - Loyalty Discount & CoffeHut Wallet  (25% Off!)
```

Reply:
I think you should add the discount amount after the prices so that users know how much they are getting off.